index.html

```diff
@@ -3,30 +3,41 @@
   <head>
     <title>"ALL about me"</title>
     <meta name="Farida" content="HTML"/>
+    <link rel="stylesheet" href = "css/index.css">
+   
   </head>
   <body>
+
+    <section id="my name" style="text-align: center;">
 <h1>Farida Hyderi</h1>
+</section>
+
     <section id="About">
 <h2>About</h2>
-<p>My name is Farida I am very entusiastic to learn new programming langusges
+<p >My name is Farida I am very entusiastic to learn new programming languages
 also very much interested in testing </p>
     </section>
 
-    <section id = "Experience">
-<h2>Experience</h2>
+    <section id = "Experience" >
+<h3>Experience</h3>
+<main>
       <ul>
 <li>Cognizant Agile Methodology from Forage </li>
 <li>Accenture Project management Forage </li>
 <li>Mastercard - Cybersecurity Job Simulation from Forage</li>
 <li>Worked on testing projects</li>
       </ul>
     </section>
+  </main>
+
+    <section id = "connect" >
+<h4>Connect</h4>
+
+
+<li><a href="www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com"> My linkedin </a></li>
+<li><a href="gmail.com"> gmail</a></li>
+<li><a href="instagram"> Instagram</a></li>
```

Comment:
For the CSS to apply to these links, their format needs to be correct. First, you'll want to wrap the list in `<ul>` tags to denote that it's a list. Second, you'll want to format each link like the below example. They need to be links that actually lead somewhere. So `instagram.com`  will need to be `https://www.instagram.com` to actually work.

```
<a href="[www.linkedin.com/in/farida-hyderi-a32700226](http://www.linkedin.com/in/farida-hyderi-a32700226)"> My LinkedIn </a>
```